code/modules/vore/eating/simple_animal_vr.dm

```diff
@@ -75,3 +75,27 @@ mob/living/simple_animal/custom_emote()
 mob/living/simple_animal/say()
 	if (away_from_players()) return
 	. = ..()
+
+/mob/living/simple_animal/attackby(var/obj/item/O, var/mob/user)
+	if (istype(O, /obj/item/weapon/newspaper) && !(ckey || (hostile && faction != user.faction)) && isturf(user.loc))
```

Comment:
You can come back and fix this at some point if you want now that they don't have to be hostile. But it's fine like this really.

Reply:
Come to think of it, we do sometimes drop hostile-but-neutral mobs in as admins, probably worth keeping the functionality for that.

Also it'll let blue shadekin newspaper the red ones, ho ho ho.